conmon/conmon.c

```diff
@@ -1366,6 +1368,24 @@ int main(int argc, char *argv[])
 		if (dup2(slavefd_stderr, STDERR_FILENO) < 0)
 			pexit("Failed to dup over stderr");
 
+		/* If LISTEN_PID env is set, we need to set the LISTEN_PID
+		   it to the new child process */
+		char *listenpid = getenv("LISTEN_PID");
+		if (listenpid != NULL) {
+			errno=0;
+			int lpid = strtol(listenpid, NULL, 10);
+			if (errno != 0 || lpid <=0)
+				pexit("Invalid LISTEN_PID %s: %s", listenpid, strerror(errno));
```

Comment:
With #1368 landed, this should be updated to `pexitf`.  and since `pexit*` includes `strerror` on it's own, you just need:

```C
pexitf("Invalid LISTEN_PID %s", listenpid);
```

Reply:
Fixed